src/helpers/GetTotalDonated.ts

```diff
@@ -1,44 +1,122 @@
 import { BigNumber, ethers } from "ethers";
 
+import { abi as gOHM } from "../abi/gOHM.json";
 import { abi as OlympusGiving } from "../abi/OlympusGiving.json";
 import { addresses } from "../constants";
-import { IBaseAddressAsyncThunk } from "../slices/interfaces";
+import { IBaseAddressAsyncThunk, IBaseAddressRecipientAsyncThunk } from "../slices/interfaces";
 
 // Calculate total amount redeemed by a user + their current redeemable balance
+/**
+ * Calculates the total amount redeemed by a user + their current redeemable balance
+ * @param address Current user's Ethereum address
+ * @param networkID ID number of the network the user is currently connected to
+ * @param provider Ethereum network provider object
+ * @returns BigNumber representation of the sOHM value of the total amount redeemed by a user
+ *          plus their current redeemable balance
+ */
 export const getTotalDonated = async ({ address, networkID, provider }: IBaseAddressAsyncThunk) => {
```

Comment:
I would suggest being explicit about the return type, so that it catches any deviations (e.g. line 20 returning "0" instead of BigNumber).

Reply:
Made this one return a string,